### PR TITLE
OSDOCS-1144: Adding OAuth token storage format to release notes

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-1144

Preview: http://file.rdu.redhat.com/~ahoffer/2020/OSDOCS-1144/release_notes/ocp-4-6-release-notes.html#ocp-4-6-security

### CODE REVIEW

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Comment by bergerhoffer** [comment]
@sttts Can you please review this release notes entry for the OAuth token storage format change?

Preview (internal): http://file.rdu.redhat.com/~ahoffer/2020/OSDOCS-1144/release_notes/ocp-4-6-release-notes.html#ocp-4-6-security

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -56,6 +56,17 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
...
+[id="ocp-4-6-security"]
+=== Security
+
+[id="ocp-4-6-oauth-token-storage"]
+==== Secure OAuth token storage format
+
+OAuth access token and OAuth authorize token object names are now stored as non-sensitive object names.
+
+Previously, secret information was used as the OAuth access token and OAuth authorize token object names. When etcd is encrypted, only the value is encrypted, so this sensitive information was not encrypted.
+
```
Comment (sttts) [inline release_notes/ocp-4-6-release-notes.adoc:69]: Note: for cluster installed prior to 4.6, ensure that old tokens (without a `sha256~` prefix) either expire eventually or are manually deleted in order to switch to a secure storage format.

Comment (bergerhoffer) [reply release_notes/ocp-4-6-release-notes.adoc:69]: Added a note on this, thanks!

**Comment by bergerhoffer** [comment]
@barleyer Can you please review this release notes update?

Preview (internal): http://file.rdu.redhat.com/~ahoffer/2020/OSDOCS-1144/release_notes/ocp-4-6-release-notes.html#ocp-4-6-security

**Pull request merged**